initapp.sh

```diff
@@ -10,21 +10,36 @@
 : "${FLASK_ENV:="development"}"
```

Comment:
this is the other file I'm wondering if we would like to migrate to python. What do you think?